apps/openmw/mwworld/inventorystore.cpp

```diff
@@ -383,7 +392,16 @@ void MWWorld::InventoryStore::autoEquip (const MWWorld::Ptr& actor)
 
         for (int j = 0; j < static_cast<int>(weaponSkillsLength); ++j)
         {
-            int skillValue = stats.getSkill(static_cast<int>(weaponSkills[j])).getModified();
+            int skillValue = 0;
+            if (actor.getClass().isNpc())
+            {
+                MWMechanics::NpcStats& stats = actor.getClass().getNpcStats(actor);
+                skillValue = stats.getSkill(ESM::Skill::Unarmored).getModified();
```

Comment:
Why did you replace `weaponSkills[j]`? I'm pretty sure I checked that against the game's assembly when I implemented it.

Reply:
Sorry, my fault. Thanks for spotting that.